llvm/test/Transforms/ConstraintElimination/implied-by-bounded-memory-access.ll

```diff
@@ -0,0 +1,272 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes=constraint-elimination -S %s | FileCheck %s
+
+@g = private unnamed_addr constant [5 x i8] c"test\00"
```

Comment:
Can you please add a negative test with an external global? To verify that this does not use minimal object size.